content/FAQ.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Frequently Asked Questions
+url: FAQ/
+summary: Frequently asked questions
+showToc: true
+tocopen: true
+---
+
+## I read the docs but still need help!
+
+Come join the [discord](https://discord.gg/PeHR6Tjyvn)! Let the community help you in real-time!
+
+## What is VPupPr
+
+VPupPr is an open-source project that animates 3D objects using the glb or vrm file extension. 
```

Comment:
> animates 3D objects using the glb or vrm file extension.

Could be reworded as: animates `glb` and `vrm` models. Backticks should be used to denote file formats.